controllers/index.js

```diff
@@ -1,10 +1,10 @@
 const router = require("express").Router();
 
 const apiRoutes = require("./api");
-// const homeRoutes = require("./home-routes.js");
+const homeRoutes = require("./home-routes.js");
 // const dashboardRoutes = require("./dashboard-routes.js");
```

Comment:
Did you mean to leave this commented out?

Reply:
nope, probably just lost track of which branch i was on